xwiki-platform-core/xwiki-platform-tag/xwiki-platform-tag-ui/src/main/resources/XWiki/TagCloud.xml

```diff
@@ -485,22 +486,36 @@ $xwiki.ssx.use('XWiki.TagCloud')##
   ## Get tag count map. Key: tag, value: number of occurrences.
   #set ($tagCount = $xwiki.tag.getTagCount("$!tagCloudSpace"))
 #end
+## Organize tags as a list of {name, count} object
+## so that it is easier to handle (sorting, limiting, ...)
```

Comment:
note that we use a 120 line size so there's no need to split this comment on 2 lines